python/ray/data/tests/test_execution_optimizer.py

```diff
@@ -518,6 +518,12 @@ def test_read_map_chain_operator_fusion_e2e(ray_start_regular_shared, enable_opt
     assert name in ds.stats()
 
 
+def test_write_operator(ray_start_regular_shared, enable_optimizer, tmp_path):
```

Comment:
can we also add a unit test for `Write` operator similar to others - such as `test_sort_operator` below.